install-binary.sh

```diff
@@ -71,7 +71,7 @@ verifySupported() {
 getDownloadURL() {
   version=$(git -C "$HELM_PLUGIN_DIR" describe --tags --exact-match 2>/dev/null || :)
   if [ -n "$version" ]; then
-    DOWNLOAD_URL="https://github.com/$PROJECT_GH/releases/download/$version/helm-diff-$OS.tgz"
+    DOWNLOAD_URL="https://github.com/$PROJECT_GH/releases/download/$version/helm-diff-$OS-$ARCH.tgz"
```

Comment:
This might be a breaking change as I thought `helm plugin install --version $VERSION` used the `plugin.yaml` and the `install-binary.sh` from HEAD of the default branch, regardless of what `$VERSION` you specified.

Perhaps we'd better have some fallback code to use `DOWNLOAD_URL="https://github.com/$PROJECT_GH/releases/download/$version/helm-diff-$OS.tgz` as before, when the `-$OS` variant was not found. WDYT?

Reply:
That makes sense. I added a conditional here based on the version and fixed the github call to get the latest download URL. Let me know if this approach is good with you.